### ISSUE TITLE
Table already exists

### ISSUE THREAD

**Issue by chap** [ricardochimal/taps#21]
Not sure if this is intentional, but I'm getting a 'table already exists' if I try to pull and overwrite my local db.

```
/Library/Ruby/Gems/1.8/gems/sequel-3.10.0/lib/sequel/adapters/mysql.rb:157:in `query': Mysql::Error: Table 'projects' already exists (Sequel::DatabaseError)
```

**Comment by ricardochimal** [comment]
taps won't remove your local tables, 0.2 used to because it was a side effect of the activerecord schema loader

**Comment by ricardochimal** [comment]
I made a few quick fixes, try taps 0.3.3. keep in mind that there is no going back when you drop the table(s)

**Comment by chap** [comment]
Awesome, thanks.
